db/migrate/20230612075547_add_kit_allocation_type_to_kit_allocations.rb

```diff
@@ -0,0 +1,10 @@
+class AddKitAllocationTypeToKitAllocations < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured do
+      create_enum :kit_allocation_inventory, ["inventory_in", "inventory_out"]
+      change_table :kit_allocations do |t|
+        t.enum :inventory, enum_type: "kit_allocation_inventory", default: "inventory_in", null: false
```

Comment:
`kit_allocation_type` was a brand new column introduced in this PR. There's no need to check in migrations that add the column, then remove it and replace it with a different one. Just change the migration itself and fix your local :)

Reply:
reused kit_allocation_type, the reason to rename to "inventory" was conventional way to give name to enum_type: "model_name_enum_name"

ie 
`# db/migrate/20131220144913_create_articles.rb
def up
  create_enum :article_status, ["draft", "published"]

  create_table :articles do |t|
    t.enum :status, enum_type: :article_status, default: "draft", null: false
  end
end
`

so it was becoming "kit_allocation_kit_allocation_type". now  just used "kit_allocation_type" bypassing that convention

Reply:
done